python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -222,6 +225,20 @@ def _get_ray_cr_with_no_tpus() -> dict:
     return cr
 
 
+def _get_ray_cr_with_only_requests() -> dict:
+    """CR contains only resource requests"""
+    cr = get_basic_ray_cr()
+
+    for container in [
+        container
+        for group in [cr["spec"]["headGroupSpec"]] + cr["spec"]["workerGroupSpecs"]
+        for container in group["template"]["spec"]["containers"]
+    ]:
+        container["resources"]["requests"] = container["resources"]["limits"]
+        del container["resources"]["limits"]
```

Comment:
We replace the `resource requests` with `resource limits` and then delete `resource limits` to make the new test data.